app.js

```diff
@@ -0,0 +1,34 @@
+let allBooks = document.querySelector('.books');
+
+document.getElementById('btn').addEventListener('click' , getBooks);
+
+
+function getBooks(){
+    // using Fetch 
+    fetch('books.json')
+    .then(function(re){
+        return re.json();
+    })
+
+    .then(function(books){
+
+        let bookDiv = '';
```

Comment:
Try to think always about isolating common code in a separate function. For example, the part where you create a DOM element for a book could be its own function. That way, you can use the same code regardless of the way you get the books from

Reply:
Aha alright, you mean to create a small function for putting the html element in it only?

Reply:
Yeah, the function creates the necessary divs and appends them to the books div element

Reply:
Ok :)